spec/migrations/20190327132620_dialog_field_load_values_on_init_spec.rb

```diff
@@ -0,0 +1,41 @@
+require_migration
+
+describe DialogFieldLoadValuesOnInit do
+  let(:dialog_field_stub) { migration_stub(:DialogField) }
+
+  migration_context :up do
+    let (:show_button) { nil }
+
+    before do
+      [true, false, nil].each do |load_values|
+        dialog_field_stub.create!(:show_refresh_button => show_button, :load_values_on_init => load_values)
+      end
+
+      migrate
+    end
+
+    context "when show_refresh_button is true" do
+      let (:show_button) { true }
```

Comment:
Pretty sure there shouldn't be a space between the `let` and `(` here. And that's why you're getting the rubocop issues.

Reply:
Same with the other `let(:show_button)` calls.